search/update_attribute_configuration.py

```diff
@@ -72,8 +72,9 @@ def update_product(product_to_update_id):
     print(updated_product)
 
     print('---Wait 5 minutes to be sure the catalog has been indexed after the changes---:')
-    time.sleep(300)
+    time.sleep(60)
```

Comment:
print says you wait 5 minutes, but sleep is for 1 min

Reply:
changed to 2 minutes